### ISSUE TITLE
WAI Task Force pages are out of date

### ISSUE THREAD

**Issue by maryjom** [w3c/wai-about-wai#148]
Is the [groups page that lists task forces](https://www.w3.org/WAI/about/groups/taskforces/) an old page that needs deleting? If it is still a valid page, it needs a bit of updating.  

- WCAG working group is now AG WG. 
- It is missing the re-established WCAG2ICT task force under the AG WG. A description of this TF is:
>The objective of the WCAG2ICT Task Force is to develop documentation describing how WCAG 2.x and its principles, guidelines, and success criteria could apply to non-Web Information and Communications Technologies (ICT).
-  Is the SVG task force still active?  The [SVG TF link](http://www.w3.org/WAI/PF/svg-a11y-tf/) goes to yet another old page that lists Rich Schwerdtfeger as a contact. He hasn't been involved in W3C for quite some time now.

I imagine that some of the other TF sub-pages are out of date as well.

**Comment by shawna-slh** [comment]
Thanks, @maryjom !
I've [asked folks to submit updates](https://lists.w3.org/Archives/Public/public-wai-cc/2022Jul/0000.html).

**Comment by maryjom** [comment]
@shawna-slh Thanks Shawn.  I have submitted a few updates (not fully comprehensive) but for WCAG2ICT and AG WG new name in the above mentioned PR. I couldn't assign anyone to review, but assume you'll get notified.
